client/src/components/molecules/VersionNumbers/VersionStatus.module.scss

```diff
@@ -19,18 +19,16 @@
 .indicatorActive {
     display: inline-block;
     margin-right: $spacer / 4;
-    margin-bottom: -.1rem;
+    margin-bottom: -0.1rem;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)